src/header.js

```diff
@@ -35,6 +39,13 @@ class Header extends Component {
             Log in
           </Link>
         </div>
+        <div class="header-right" id="logout">
+          <Form onSubmit={logoutFirebase}>
```

Comment:
Maybe we can pull this out of the form using the button onclick caller... something to look into

Reply:
Good call. It's been done in a recent commit.